tests/generators/test_rust_generator.py

```diff
@@ -16,16 +16,23 @@ def test_constants_are_declared_correctly():
         const i8 foo = 17;
         // Comment
         const u16 bar = 0x42;
+        // Large constant
+        const u64 foobar = 1000000000;
         }
     """, RustGenerator, """
 /// There is some documentation about foo.
 pub const FOO: i8 = 17;
 """,
-"""
+                           """
```

Comment:
This is intended?

Reply:
I'd say it's the same reasoning as `_escape_rust_keywords`, I tried to look in the documentation of `jinja2` but there's no convention. I'd rather keep it as-is, as filters are usually no-op in some case but aren't prefixed with `try_`.